src/content/patterns/adding-creating-pattern/adding-creating-pattern.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: Adding or Creating
+---
+
+
+<ImageComponent cols="8">
+
+![Example](images/1.jpg)
+
+</ImageComponent>
+
+*Patterns for Creating and Adding are identical, the difference is
+the context in which they are used. It is important all 'Create' words are
+replaced with 'Adding' and vice versa.*
+
+#### Maintainers:
+
+[Vikki Paterson](https://github.ibm.com/VICTORIA), [Arnaud Gillard](https://github.ibm.com/Arnaud-Gillard)
+
+#### Status:
+
+Experimental
+
+## Options
+
+<AnchorLinks small>
+
+- [Full page](#full-page)
+- [Side panel](#side-panel)
+- [Modal](#modal)
+
+</AnchorLinks>
+
+
+## Full page
+
+If there are more than 2 options involved in the creation process a full page modal / panel, overlaying the content where the creation was initiated should be used. 
+
+The creation of resources can vary considerably in complexity and with this in mind there are a number of alternatives to consider when designing a creation flow.
+
+<AnchorLinks small>
+
+- [Advanced Option](#advanced-option)
+- [Stepped creation](#stepped-creation)
+- [Illustration or Image](#illustration-or-image)
+- [Single scrollable page](#single-scrollable-page)
+- [Button position](#button-position)
+
+</AnchorLinks>
+
+<ImageComponent cols="8">
+
+![Example](images/2.jpg)
+
+</ImageComponent>
+
+
+### Advanced Option
+
+Consider having an advanced option to avoid overwhelming users, while still allowing more advanced users to access everything.
+
+In situations where there are large numbers of options consider, by default, having only a minimal set visible. Provide the ability for users to chose if they want to expose more advanced, optional, options. Users should always be able to get back out of advanced.
+
+It is recommended that if the user switches to the advanced mode this is remembered and the next time they enter the same creation process it starts in advanced .
+
+<ImageComponent cols="4">
+
+![Example](images/3.jpg)
+
+</ImageComponent>
+<br />
+<ImageComponent cols="8">
+
+![Example](images/4.jpg)
+
+</ImageComponent>
+
+Take into consideration that even if by default there are only a few options visible, you might want the ability to expose additional advanced options meaning more space would be required.
+
+
+### Stepped creation
+Consider whether breaking the process up into steps could help the user. 
+If the creation task is complex or involves a large number of mandatory options, think about:
+- Displaying related options together
+- Grouping dependent options together
+- Using the space to provide additional information / explanatory text for the option(s) 
+- Using this as a simple stepped version with minimal options and extra guidance alongside the advanced option to expose all options 
+
+<ImageComponent cols="8">
+
+![Example](images/5.jpg)
+
+</ImageComponent>
+
+
+### Illustration or Image
```

Comment:
This doesn't seem like on the same level as "stepped creation" and  "single scrollable page" maybe make it a H4 (####) instead of H3 (###).